PurchasesCoreSwift/Misc/SystemInfo.swift

```diff
@@ -17,7 +23,7 @@ import WatchKit
 import AppKit
 #endif
 
-@objc(RCSystemInfo) open class SystemInfo: NSObject {
+open class SystemInfo {
```

Comment:
open is probably not needed nor wanted, right?

Reply:
💯